installer/packer/scripts/kov/start_kov.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
```

Comment:
This file seems unnecessary if you produce an env file and put it in /etc/default/kovd